ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/deploy_container/scripts/dockerCPU.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+# Usage - Script takes one argument, the number of cpus for the docker container
+# ie bash dockerCPU.sh 4
+# Argument cannot be 0 or greater than the number of CPUs on the machine
+# If /var/log/docker_cpu does not exist, run: echo 0 > /var/log/docker_cpu
+
+set -eu
+
+cpu_limit=$1
+start_cpu=$(tail -n 1 /var/log/docker_cpu)
+max_cpus=$(lscpu | grep "CPU(s):" | head -n 1 | awk '{split($0,a," "); print a[2]}')
+cpus_to_use=""
+
+if [ $cpu_limit -gt $max_cpus ] || [ $cpu_limit == "0" ]
+then
+    echo "ERROR: The container cannot have more cpus than there are cpus on the host or be 0"
+    exit 1
+fi
+
+for i in $(seq 0 $(($cpu_limit - 1)));
+do
+    cpus_to_use="$cpus_to_use,$((($i + $start_cpu) % $max_cpus))"
```

Comment:
Nice fun bit of code ;-) I'd be tempted to move the `cpus_to_use=""` initialisation line just before here to keep it all together in the file